internal/cmd/server/list.go

```diff
@@ -24,11 +25,35 @@ var ListCmd = base.ListCmd{
 
 	DefaultColumns: []string{"id", "name", "status", "ipv4", "ipv6", "private_net", "datacenter", "age"},
 
-	Fetch: func(ctx context.Context, client hcapi2.Client, _ *pflag.FlagSet, listOpts hcloud.ListOpts, sorts []string) ([]interface{}, error) {
+	AdditionalFlags: func(cmd *cobra.Command) {
+		cmd.Flags().StringSlice("status", nil, "Only servers with one of these statuses are displayed")
```

Comment:
Can we add autocomplete for the available statuses?

Reply:
Good catch 🙂 Added in d863461168c8e5452cc4101e5744660e31660882